File: examples/python/organization.py
import requests
from token import get_access_token

API_URL = 'https://api.3plguys.com/v0/organization'

def get_organization(token):
    """
    Fetches organization details from the 3PLGuys Public API.
    """
    headers = {'Authorization': f'Bearer {token}'}
    res = requests.get(API_URL, headers=headers)

    if res.status_code == 200:
        org = res.json()
        print(f"🏢 {org['name']}")
        print(f"Website: {org['website']}")
        print(f"Contact: {org['contact']['name']} | {org['contact']['phoneNumber']}")
        addr = org['address']
        print(f"Address: {addr['addressline1']}, {addr['city']}, {addr['state']} {addr['zipcode']}")
    else:
        print(f"❌ Org error {res.status_code}: {res.text}")


if __name__ == '__main__':
    token = get_access_token()
    if token:
        get_organization(token)

File: examples/nodejs/organization.js
const axios = require('axios');
const { getAccessToken } = require('./token');

async function getOrganization() {
  const token = await getAccessToken();
  if (!token) return;

  try {
    const res = await axios.get('https://api.3plguys.com/v0/organization', {
      headers: { Authorization: `Bearer ${token}` }
    });

    const org = res.data;
    console.log(`🏢 ${org.name}`);
    console.log(`Website: ${org.website}`);
    console.log(`Contact: ${org.contact.name} | ${org.contact.phoneNumber}`);
    console.log(`Address: ${org.address.addressline1}, ${org.address.city}, ${org.address.state} ${org.address.zipcode}`);
  } catch (err) {
    console.error('❌ Org error:', err.response?.status, err.response?.data || err.message);
  }
}

getOrganization();
